app/src/main/res/layout/recently_played_activity.xml

```diff
@@ -19,14 +19,13 @@
         android:id="@+id/recently_played_toolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/colorPrimary"
+        android:background="@color/component_color_shared_activity_toolbar_color"
         android:fontFamily="sans-serif"
         android:minHeight="?attr/actionBarSize"
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator"
         app:title="@string/recently_played_activity"
-        app:titleTextAppearance="@style/ToolbarTextAppearance"
-        app:titleTextColor="@color/color_def_white" />
+        app:titleTextAppearance="@style/ToolbarTextAppearance"/>
```

Comment:
If you are using this style then make sure that in `styles.xml` for `ToolbarTextAppearance` you use the correct toolbar color. Currently its set to `color_def_white` which is incorrect. It should be `component_color_shared_activity_toolbar_text_color`

Reply:
Okay that also we do for all colors in style. We have to document this also in Post PR changes.
I see you already added this thanks.